go-controller/pkg/util/net_linux.go

```diff
@@ -170,27 +171,36 @@ func LinkNeighExists(link netlink.Link, neighIP net.IP, neighMAC net.HardwareAdd
 	return false, nil
 }
 
-func DeleteConntrack(ip string, port int32) {
+func DeleteConntrack(ip string, port int32, protocol kapi.Protocol) {
```

Comment:
@JacobTanenbaum Can we just have DeleteConntrack return an error, and the caller can print it, rather than klogging inside DeleteConntrack?

Reply:
done